USEPA/mywaterway
# Bugfix/assessments missing from gis and attains

## Related Issues:
* https://app.breeze.pm/projects/100762

## Main Changes:
* Adds support for Assessments that are missing from the GIS but are not found in the ATTAINS Assessment Units and/or Assessments services.

## Steps To Test:
1. Test the following locations. These are locations where the Assessments service has no data for the assessment IDs missing from the GIS. But the Assessment Units service does have data including the Org Name, Org Type, Org ID, and Assessment Name.
http://localhost:3000/community/san%20diego%20ca/overview
http://localhost:3000/community/fair%20oaks%20ca/overview
http://localhost:3000/community/27127,%20Winston%20Salem,%20North%20Carolina/overview

2. Verify a GA event is being logged using GA Debug Chrome extension or similar: "The Assessments and GIS services did not return data for the following assessment IDs..."
3. On line 396, add a line 397 with the content `res.items = [];`. This tests how the app handles Assessments being missing from both the Assessments and Assessment Units services. Re-test a location from above. The missing assessments should now have names of "Unknown" and the Waterbody Report links hidden.


## TODO:
- [ ] Add Cypress tests for the following scenarios:
- Where the Assessment Units service data does not include data for 1 or more of the Assessment IDs missing from the GIS services.
- Where the Assessments service does not include data for 1 or more of the Assessment IDs missing from the GIS services.
